src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
If playlist is sort of an afterthought for the user, it's probably fine. I coming from the perspective of people wanting to create a playlist player and then not being able to see how to do that. I don't work with this often, so I'm fairly unfamiliar, which maybe makes me a good user test target? not sure. I honestly didn't realize the playlist option was just below the episode selection. When the interface changed, I assumed that's when the playlist feature became available to me. I didn't scroll down to find it before making selections.

I think it should only show two iframe options at a time based on playlist option rather than all options. I feel like users won't understand which one to use even if we explain it in full detail. Showing all iframe options would also make the preview confusing, how would I know which one is being previewed and does it preview the playlist when selected?